src/bio.json

```diff
@@ -1,12 +1,11 @@
 {
     "bio": [
-        "Ryan Edgar is a Digital Business Integration Associate within Accenture ",
-        "Digital for Front-End Development North America.",
+        "Ryan Edgar is a Digital Business Integration Associate within Accenture Digital for Front-End Development North America.",
         " She recently transitioned from a career in marketing and advertising into the tech industry.",
```

Comment:
Remove the leading or trailing spaces from this data and add the space in the JS.  Your solution is going to change, but this can be accomplished the way you're currently solving this just by changing this `${bio[0]}${bio[1]}${bio[2]}` to `${bio[0]} ${bio[1]} ${bio[2]}` (notice the spaces are just being added within the string)